doc/source/development/extending.rst

```diff
@@ -50,8 +50,9 @@ decorate a class, providing the name of attribute to add. The class's
 
 Now users can access your methods using the ``geo`` namespace:
 
-      >>> ds = pd.DataFrame({'longitude': np.linspace(0, 10),
-      ...                    'latitude': np.linspace(0, 20)})
+      >>> ds = pd.Dataframe(
+      ...   {"longitude": np.linspace(0, 10), "latitude": np.linspace(0, 20)}
```

Comment:
Might be under-indented by a space

Reply:
Two spaces right? Since the leading space is the space between "..." and the start of code.

Reply:
Yeah looks like it has only two (should be four)